### PR TITLE
[ODS-6082] Add PostmanTest coverage for role named program in programEvaluation domain

### PR DESCRIPTION
**Pull request by semalaiappan**
…n domain

### CODE REVIEW

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 1]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"    pm.test(\"Status code is 201\", () => {\r",
+															"        pm.expect(pm.response.code).to.equal(201);\r",
+															"    });\r",
+															"    \r",
+															"\r",
+															"pm.environment.set('known:programevaluation:id', pm.response.headers.one('Location').value.split(\"/\").pop());\r",
+															"\r",
+															"var requestJson = JSON.parse(request.data);\r",
+															"pm.environment.set('known:programEvaluationTitle', requestJson.programEvaluationTitle);\r",
+															"console.log(requestJson.programEvaluationTitle);\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2482]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 2]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"if (pm.environment.get(\"known:standardVersion\")[0] != \"4.0.0\") {\r",
+															"    pm.test(\"Status code is 201\", () => {\r",
+															"        pm.expect(pm.response.code).to.equal(201);\r",
+															"    });\r",
+															"    \r",
+															"pm.environment.set('known:programEvaluationObjective:id', pm.response.headers.one('Location').value.split(\"/\").pop());\r",
+															"\r",
+															"var requestJson = JSON.parse(request.data);\r",
+															"pm.environment.set('known:programEvaluationObjectiveTitle', requestJson.programEvaluationObjectiveTitle);\r",
+															"console.log(requestJson.programEvaluationObjectiveTitle);\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2622]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 3]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"if (pm.environment.get(\"known:standardVersion\")[0] != \"4.0.0\") {\r",
+															"    pm.test(\"Status code is 201\", () => {\r",
+															"        pm.expect(pm.response.code).to.equal(201);\r",
+															"    });\r",
+															"    \r",
+															"pm.environment.set('known:programEvaluationElement:id', pm.response.headers.one('Location').value.split(\"/\").pop());\r",
+															"\r",
+															"var requestJson = JSON.parse(request.data);\r",
+															"pm.environment.set('known:programEvaluationElementTitle', requestJson.programEvaluationElementTitle);\r",
+															"console.log(requestJson.programEvaluationElementTitle);\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2762]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 4]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"    pm.test(\"Status code is 201\", () => {\r",
+															"        pm.expect(pm.response.code).to.equal(201);\r",
+															"    });\r",
+															"\r",
+															"pm.environment.set('known:evaluationRubricDimension:id', pm.response.headers.one('Location').value.split(\"/\").pop());\r",
+															"\r",
+															"var requestJson = JSON.parse(request.data);\r",
+															"pm.environment.set('known:evaluationRubricRating', requestJson.evaluationRubricRating);\r",
+															"pm.environment.set('known:evaluationCriterionDescription', requestJson.evaluationCriterionDescription);\r",
+															"console.log(requestJson.evaluationRubricRating);\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2903]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 5]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"console.log(requestJson.evaluationCriterionDescription);\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2904]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 6]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"if (pm.environment.get(\"known:standardVersion\")[0] != \"4.0.0\") {\r",
+															"    pm.test(\"Status code is 200\", () => {\r",
+															"        pm.expect(pm.response.code).to.equal(200);\r",
+															"    });\r",
+															"\r",
+															"    pm.test(\"Should return some items for verification\", () => {\r",
+															"        const responseItems = pm.response.json();\r",
+															"        pm.expect(responseItems.evaluationRubricRating).to.equal(pm.environment.get('known:evaluationRubricRating'));\r",
+															"        pm.expect(responseItems.evaluationCriterionDescription).to.equal(pm.environment.get('known:evaluationCriterionDescription'));        \r",
+															"        console.log(pm.environment.get('known:evaluationRubricRating'));\r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2955]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 7]
```diff
@@ -2254,215 +2219,1398 @@
...
+															"        console.log(pm.environment.get('known:evaluationCriterionDescription'));        \r",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:2956]: remove console.log

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json [hunk 8]
```diff
@@ -1,53 +1,18 @@
 {
 	"info": {
-		"_postman_id": "17588d9d-936f-4ddd-ad10-a65bda915841",
+		"_postman_id": "f10b065d-408d-4dab-b8b0-7712923d69e1",
```
Comment (simpat-jesus) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json:3]: changes inside "info" can be reverted, those are values assigned by postman based on the environment/user that exported the collection

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

**Comment by semalaiappan** [comment]
@simpat-jesus  Fixed all review comments

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**